tests/test-policies/src/test/java/org/mule/test/policy/PolicyTestCase.java

```diff
@@ -134,4 +158,19 @@ public void start() {
       }
     }
   }
+
+  public static class DisposeListenerMessageProcessor implements Processor, Disposable {
```

Comment:
is this related to this pr or just some conflicts in git?

Reply:
It's related to a rebase conflict I had. But in one of the last commits I restored it as it is in mule-4.x.